tests/scripts/check-api-docs.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
Would it be better name this script `verify-apidoc.sh` to consistent with other verify scripts? And as said above, just add a `verify-apidoc` under make verify https://github.com/kubeedge/kubeedge/blob/master/Makefile#L67 is enough.

Reply:
thanks for the review! I will make the changes

Reply:
Can I just implement the changes to the generated file if it needs to be generated after verifying or just leave it at verification?